compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -1867,6 +1867,9 @@ void OMR::X86::CodeGenerator::doRegisterAssignment(TR_RegisterKinds kindsToAssig
 
       self()->doBackwardsRegisterAssignment(kindsToAssign, self()->getAppendInstruction());
       }
+
+   if (TR::Options::getCmdLineOptions()->getOption(TR_MoveOOLInstructionsToWarmCode))
```

Comment:
In your previous commit this feature was done unconditionally (on by default).  Now it is only enabled with this option.  Is that your intent?   I suppose downstream projects could enable the option programmatically if they want this behaviour by default.

Apologies, but I may not have been clear in my suggestion that you could leave the behaviour you had as default, but provide an option to disable it if necessary.

Reply:
In the future, `TR_EnableCodeCacheDisclaiming `might imply `TR_MoveOOLInstructionsToWarmCode`.